sled-agent/src/services.rs

```diff
@@ -3889,9 +3894,13 @@ impl ServiceManager {
 
         for port_config in &our_ports {
             for addr in &port_config.addrs {
-                info!(self.inner.log, "configuring port: {port_config:?}");
+                let prop_name = format!("uplinks/{}_0", port_config.port);
+                info!(
+                    log,
+                    "Adding new property to uplinkd service. Name: {prop_name} Value: {}", addr.to_string(),
```

Comment:
Nit - I think I'd use explicit slog key/values for `name` and `value` instead of squashing them into the main log string.